ee/clickhouse/sql/trends/breakdown.py

```diff
@@ -56,7 +56,9 @@
     )
     GROUP BY day_start, breakdown_value
     ORDER BY breakdown_value, day_start
-) GROUP BY breakdown_value
+)
+GROUP BY breakdown_value
+ORDER BY breakdown_value
```

Comment:
I'm guessing a breakdown_value might be something like "Safari" ?  I think that makes sense to have as ASC.

I guess alternatively ordering by the largest count might be relevant but not sure what that should mean when we have a groupArray.

Can we add the trends owner for review?

Reply:
The ordering is undefined/irrelevant for the client right now - we're only ordering here for the tests, so breakdown_value imo makes sense.